Dockerfile

```
@@ -0,0 +1,47 @@
+# Stage 1: Base Node.js environment
+FROM node:18-alpine AS base
+
+# Set the working directory
+WORKDIR /app
+
+# Copy package manager lock files
+COPY package.json bun.lockb /app/
+
+# Install Bun for faster dependency installation
+RUN npm install -g bun && bun install --frozen-lockfile
+
+# Copy the entire application code
+COPY . .
+
+# Stage 2: Build the application
+FROM base AS builder
+
+# Set environment variables for production
+ENV NODE_ENV=production
+
+# Build the Next.js application
+RUN bun run build
+
+# Remove development dependencies
+RUN bun install --production --frozen-lockfile
+
```

add command for prisma generate
```suggestion
RUN bunx prisma generate
```